src/build-preview.ps1

```diff
@@ -0,0 +1 @@
+dotnet build -c Release '-p:GeneratePackages=true;VersionSuffix=preview.1' ./log4net/log4net.csproj
```

Comment:
zarro can do this, though not with a set suffix (it will generate one from the current date/time and sha) - but I should add support for named versions. This is fine for now (: